inc/inventory/request.class.php

```diff
@@ -45,19 +46,31 @@
  */
 class Request
 {
-    const DEFAULT_FREQUENCY = 24;
-
-    const XML_MODE    = 0;
-    const JSON_MODE   = 1;
-
-    const PROLOG_QUERY = 'PROLOG';
-    const INVENT_QUERY = 'INVENTORY';
-    const SNMP_QUERY   = 'SNMP';
-    const OLD_SNMP_QUERY   = 'SNMPQUERY';
-
-    const COMPRESS_NONE = 0;
-    const COMPRESS_ZLIB = 1;
-    const COMPRESS_GZIP = 2;
+   const DEFAULT_FREQUENCY = 24;
+
+   const XML_MODE    = 0;
+   const JSON_MODE   = 1;
+
+   const PROLOG_QUERY = 'PROLOG';
+   const INVENT_QUERY = 'INVENTORY';
+   const SNMP_QUERY   = 'SNMP';
+   const OLD_SNMP_QUERY   = 'SNMPQUERY';
+
+   const CONTACT_ACTION = 'contact';
+   const REGISTER_ACTION = 'register';
+   const CONFI_ACTION = 'configuration';
+   const INVENT_ACTION = 'inventory';
+   const NETDISCOVERY_ACTION = 'network-discovery';
+   const NETINV_ACTION = 'network-inventory';
```

Comment:
I forgot this point. Action strings should be "netdiscovery" and "netinventory" as to be coherent with the task name on agent side.